tests/functional.mk

```diff
@@ -8,8 +8,21 @@ FTST_IMAGE := $(FTST_TMPDIR)/rkt-inspect.aci
 FTST_IMAGE_MANIFEST_SRC := $(MK_SRCDIR)/image/manifest
 FTST_IMAGE_MANIFEST := $(FTST_IMAGE_DIR)/manifest
 FTST_IMAGE_TEST_DIRS := $(FTST_IMAGE_ROOTFSDIR)/dir1 $(FTST_IMAGE_ROOTFSDIR)/dir2
+FTST_ACE_MAIN_IMAGE_DIR := $(FTST_TMPDIR)/ace-main
+FTST_ACE_MAIN_IMAGE_ROOTFSDIR := $(FTST_ACE_MAIN_IMAGE_DIR)/rootfs
+FTST_ACE_MAIN_IMAGE := $(FTST_TMPDIR)/rkt-ace-validator-main.aci
+FTST_ACE_MAIN_IMAGE_MANIFEST_SRC := Godeps/_workspace/src/github.com/appc/spec/ace/image_manifest_main.json
+FTST_ACE_MAIN_IMAGE_MANIFEST := $(FTST_ACE_MAIN_IMAGE_DIR)/manifest
+FTST_ACE_SIDEKICK_IMAGE_DIR := $(FTST_TMPDIR)/ace-sidekick
+FTST_ACE_SIDEKICK_IMAGE_ROOTFSDIR := $(FTST_ACE_SIDEKICK_IMAGE_DIR)/rootfs
+FTST_ACE_SIDEKICK_IMAGE := $(FTST_TMPDIR)/rkt-ace-validator-sidekick.aci
+FTST_ACE_SIDEKICK_IMAGE_MANIFEST_SRC := Godeps/_workspace/src/github.com/appc/spec/ace/image_manifest_sidekick.json
+FTST_ACE_SIDEKICK_IMAGE_MANIFEST := $(FTST_ACE_SIDEKICK_IMAGE_DIR)/manifest
 FTST_INSPECT_BINARY := $(FTST_TMPDIR)/inspect
 FTST_ACI_INSPECT := $(FTST_IMAGE_ROOTFSDIR)/inspect
+FTST_ACE_BINARY := $(FTST_TMPDIR)/ace-validator
+FTST_ACI_MAIN_ACE_BINARY := $(FTST_ACE_MAIN_IMAGE_ROOTFSDIR)/ace-validator
+FTST_ACI_SIDEKICK_ACE_BINARY := $(FTST_ACE_SIDEKICK_IMAGE_ROOTFSDIR)/ace-validator
```

Comment:
The `FTST_ACI_MAIN_ACE_BINARY` and `FTST_ACI_SIDEKICK_ACE_BINARY` are not used at all, so please remove them.